src/participants.md

```diff
@@ -5,39 +5,56 @@ permalink: /participants/
 include: true
 ---
 
-## Principal Investigators:
+### Principle Investigator:
 
-- Michael Penn, Department of Religious Studies, Stanford University
+- Michael Penn, Department Religious Studies, Stanford University
+
+
+### Associate Principle Investigators:
 - Nicholas Howe, Department of Computer Science, Smith College
 - R. Jordan Crouser, Department of Computer Science, Smith College
 
 
-## Project Consultants:
+### Stanford Libraries, CIDR Development Team:
+- Vijoy Abraham
+- Scott Bailey
+- Peter Broadwell 
+- Javier de la Rosa
+- Simon Jon Wiles
+
+
+### Stanford Libraries, Stanford Digital Depository:
+- Benjamin Albritton
 
+
+### Project Consultants:
 - C.M. Chin, Department of Classics, UC Davis
-- Clare Woods, Department of Classics, Duke University
-- Adam Bremer-McCollum, Department of Theology, Notre Dame University
 - Ayda Kaplan, Centre d’études sur les chrétiens d’Orient asbl
+- Adam McCollum, Department of Theology, Notre Dame University
+- Clare Woods, Department of Classics, Duke University
 
 
-## Project Coordinators:
-
-- Philip Abbot
+### Project Coordinators:
+- Phillip Abbot
 - Kristina Bush
 - Caitlin Rajala
 - Julia Spector
 - Shuangxia Wu
 
 
-## Research Assistants:
+### Web Design:
+- Scott Bailey
+- Stefan Hodges-Kluck
+
 
+### Research Assistants:
 - Hanna Baptist
 - Christine Barney
+- Minyue Dai
```

Comment:
Is there some reason why this particular name should not be in alphabetical order by surname?

Reply:
Not sure. I'll alphabetize it for now.